docs/_snippets/_clickstack_tagging.mdx

```diff
@@ -0,0 +1,28 @@
+import Image from '@theme/IdealImage';
+import clickstackTagsSearch from '@site/static/images/use-cases/observability/clickstack-tags-search.png';
+import clickstackTagsDashboard from '@site/static/images/use-cases/observability/clickstack-tags-dashboard.png';
+
+You can add tags to dashboards and saved searches to help organize them. Tags provide a flexible way to categorize and filter based on your needs.
+
+#### How tags work
+
+- **Organization**: Tags appear in the left sidebar, where dashboards and saved searches are grouped by their assigned tags
+- **Multiple tags**: You can add one or more tags to a single item for better categorization
+- **Auto-creation**: If you assign a tag that doesn't exist yet, it will be automatically created
+- **Easy management**: You can add or remove tags at any time to adjust your organization structure
+
+This makes it easy to find related items and maintain an organized workspace as your collection grows.
+
+<div style={{display: 'flex', justifyContent: 'flex-start', marginTop: '-20px'}}>
+  <div style={{maxWidth: '600px'}}>
+    <Image img={clickstackTagsSearch} alt="Tags in saved search" size="md" border />
+  </div>
+</div>
+
+You can also select multiple tags to filter and view items across different categories:
+
+<div style={{display: 'flex', justifyContent: 'flex-start', marginTop: '-20px'}}>
+  <div style={{maxWidth: '600px'}}>
+    <Image img={clickstackTagsDashboard} alt="Multiple tags selected in dashboard" size="md" border />
+  </div>
+</div>
```

Comment:
The styling looks a little strange when the snippet is imported into other pages. See https://clickhouse-docs-rkc69392f-clickhouse.vercel.app/docs/use-cases/observability/clickstack/search#tagging

if you want to add some margin between the text you can add a `<br/>` above the image, should do the trick.